lib/sprockets/sass/v3/importer.rb

```diff
@@ -194,6 +194,8 @@ def filter_all_processors(processors)
         end
 
         def evaluate_path_from_context(context, path, processors)
+          less_processor = Sprockets::Sass::Utils.get_less_class_by_version('LessTemplate')
```

Comment:
Yeah I understand, what I mean is that we shouldn't even attempt to the less processor if the path doesn't include `.less`, something like:

``` ruby
if path.include?('.less')
  less_processor = Sprockets::Sass::Utils.get_less_class_by_version('LessTemplate')
  processors = [less_processor] if less_processor
end
```

It might be premature optimization, but using Less inside Sass isn't standard, and we should try to limit the impact of this change for users who only use Sass